### ISSUE TITLE
Add Android target [Moved from old repo]

### ISSUE THREAD

**Issue by chippmann** [utopia-rise/godot-kotlin#3]
**Describe the problem or limitation you are having in your project:**

Godot projects can be exported to Android. 
GDNative also supports this target.
For now this project cannot export to Android.

**Describe how this feature / enhancement will help you overcome this problem or limitation:**

It will allow me to export kotlin godot's native to android.

**Show a mock up screenshots/video or a flow diagram explaining how your proposal will work:**

1) Add target in gradle build files for Android
2) Build sample project for Android
3) Test on android device

**Comment by piiertho** [comment]
I limit the dev to 64 bits platforms for now, as I'm having trouble with 32 bits because we use Long
```
e: /Users/piertho/git/kotlin-godot-wrapper/wrapper/godot-library/src/main/kotlin/godot/registration/Registration.kt: (88, 25): Type mismatch: inferred type is ULong but size_t /* = UInt */ was expected
e: /Users/piertho/git/kotlin-godot-wrapper/wrapper/godot-library/src/main/kotlin/godot/registration/Registration.kt: (92, 83): Type mismatch: inferred type is ULong but size_t /* = UInt */ was expected
e: /Users/piertho/git/kotlin-godot-wrapper/wrapper/godot-library/src/main/kotlin/godot/registration/Registration.kt: (96, 78): Type mismatch: inferred type is ULong but size_t /* = UInt */ was expected
e: /Users/piertho/git/kotlin-godot-wrapper/wrapper/godot-library/src/main/kotlin/godot/registration/Tools.kt: (57, 57): Type mismatch: inferred type is ULong but size_t /* = UInt */ was expected
e: /Users/piertho/git/kotlin-godot-wrapper/wrapper/godot-library/src/main/kotlin/godot/registration/Tools.kt: (107, 57): Type mismatch: inferred type is ULong but size_t /* = UInt */ was expected
```

**Comment by piiertho** [comment]
solved by #24
